### PR TITLE
show « check now » button on PluginManager updates and available pages

### PR DESCRIPTION
**Pull request by ndeloof**

### CODE REVIEW

**Comment by ndeloof** [comment]
@reviewbybees

**Comment by stephenc** [comment]
:+1:

**Comment by tfennelly** [comment]
:+1:

File: core/src/main/resources/hudson/PluginManager/advanced_bg.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # This file is under the MIT License by authors
 
-Check\ now=\u041F\u0440\u043E\u0432\u0435\u0440\u0438 \u0441\u0435\u0433\u0430
```
Comment (jglick) [inline core/src/main/resources/hudson/PluginManager/advanced_bg.properties]: BTW not sure if you found it or not, but `core/move-l10n.groovy` is the recommended easy way to move bundle keys with localizations.

File: core/src/main/resources/hudson/PluginManager/table.jelly [hunk 2]
```diff
@@ -147,6 +147,7 @@ THE SOFTWARE.
                 <span style="margin-left:2em;"></span>
               </j:if>
               <f:submit value="${%Download now and install after restart}" />
+              <st:include page="check.jelly"/>
```
Comment (jglick) [inline core/src/main/resources/hudson/PluginManager/table.jelly]: Does this not also show it on the _Installed_ page, where it does not make much sense?

Comment (ndeloof) [inline core/src/main/resources/hudson/PluginManager/table.jelly]: no, installed doesn't use the table tag
https://github.com/jenkinsci/jenkins/blob/7cdb85d09c0a709c59694de7b009cb2d0117578b/core/src/main/resources/hudson/PluginManager/installed.jelly#L138-138

File: core/src/main/resources/hudson/PluginManager/advanced_bg.properties (hunk 1 above)
Comment (ndeloof) [inline core/src/main/resources/hudson/PluginManager/advanced_bg.properties]: Ah nice to know, had to write my own that was ugly.

**Comment by bpedersen2** [comment]
it seems that the button does not what it is supposed to todo on the 'plugins update' page. 
There are now two buttons with name 'Submit', and so the button really triggers the plugin update, instead of fetch the  available updates.

**Comment by ndeloof** [comment]
indeed, this creates nested <form> elements. Need to revisit.

**Comment by jglick** [comment]
> Need to revisit.

So is there a confirmed regression? Is it filed? Being fixed?

**Comment by daniel-beck** [comment]
> Is it filed?

Independently filed as [JENKINS-27436](https://issues.jenkins-ci.org/browse/JENKINS-27436)

**Pull request merged**